### PR TITLE
feat:Quiz and Rules Screen

### PR DESCRIPTION
**Pull request by TheAmlanSaikia**
Questions
Rules Page

Added the Quiz page and Rules page with functionality.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -39,4 +40,4 @@
 
 button {
   font-size: calc(10px + 2vmin);
-}
+}  */
```
Comment (ravikumar0403) [inline src/App.css]: you can delete unused code

File: src/App.jsx [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
 function App() {
 
   return (
     <div className="App">
-      <Home />
+      <Navigation />
+      <Routes>
+       <Route path='/' element={<Home />} />
+       <Route path='/rules' element={<Rules />} />
+       <Route path='/quiz' element ={<QuizPage />} />
```
Comment (aniketsingh13) [inline src/App.jsx]: you can make seperate routes for router

File: src/App.jsx [hunk 3]
```diff
@@ -1,12 +1,21 @@
 import { useState } from 'react'
+import { Route, Routes } from 'react-router-dom'
 import './App.css'
+import { Navigation } from './components'
 import { Home } from './pages/Home/Home'
+import { QuizPage } from './pages/Quiz Page/Quiz'
+import { Rules } from './pages/Rules/Rules'
```
Comment (ravikumar0403) [inline src/App.jsx]: Use barrel imports to make your code cleaner

File: src/App.jsx [hunk 4]
```diff
@@ -1,12 +1,21 @@
...
 
 function App() {
```
Comment (ravikumar0403) [inline src/App.jsx:7]: This can be changed to es6 syntax

**Review by aniketsingh13 — APPROVED** [review]
code look good ui also look good to merge

File: src/data/Rules data.jsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  {
+    id: 4,
+    rule: "There are no deductions for wrong answers.",
+  },
+  {
+    id: 5,
+    rule: "To go to next question, click on Next",
+  },
+];
+export { RulesData };
```
Comment (ravikumar0403) [inline src/data/Rules data.jsx:23]: avoid using spaces in file names..

**Review by ravikumar0403 — APPROVED** [review]

**Pull request merged**